x-pack/plugins/rule_registry/server/plugin.ts

```diff
@@ -33,17 +66,45 @@ export class RuleRegistryPlugin implements Plugin<RuleRegistryPluginSetupContrac
       },
     });
 
-    service.init().catch((originalError) => {
+    ruleDataService.init().catch((originalError) => {
       const error = new Error('Failed installing assets');
       // @ts-ignore
       error.stack = originalError.stack;
       logger.error(error);
     });
 
-    return service;
+    return { ruleDataService };
   }
 
-  public start(): RuleRegistryPluginStartContract {}
+  public start(
+    core: CoreStart,
+    plugins: RuleRegistryPluginStartDependencies
+  ): RuleRegistryPluginStartContract {
+    this.eventLogService = new EventLogService({
+      config: {
+        indexPrefix: this.config.index,
+        isWriteEnabled: this.config.write.enabled,
+      },
+      dependencies: {
+        clusterClient: Promise.resolve(core.elasticsearch.client), // TODO: get rid of Promise
+        spaces: plugins.spaces.spacesService,
+        logger: this.logger.get('eventLog'),
+      },
+    });
+
+    // TODO: remove before merge
+    testEventLogImplementation(this.eventLogService, this.logger);
```

Comment:
Not sure what the intent of this is. Could it be written instead as an integration test that we can keep around?